cdap-proto/src/main/java/io/cdap/cdap/proto/operationrun/OperationMeta.java

```diff
@@ -29,6 +29,7 @@
  */
 public class OperationMeta {
   private final Set<OperationResource> resources;
+  @Nullable
```

Comment:
why is this nullable? Shouldn't it always have a start time?

Reply:
Fixed